Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ruby:2.0.0-p643
+
+# Install deps
+RUN apt-get update \
+  && apt-get install -y libreadline-dev nano vim \
+  && apt-get purge -y --auto-remove
+
+# Ensure Gemfile.lock is up to date
+RUN bundle config --global frozen 1
+
+# Install latest released Napa to get baseline dependencies
+RUN gem install napa
```

Comment:
Yes, but this provides a baseline layer of cache specific to Napa. This way future changes to Gemfile and Gemfile.lock still build faster. It could theoretically allow someone to get started with fig/docker-compose really quickly by running `napa new` in the container. I haven't messed with that yet, as development in containers is still a bit of a pain.